src/main/java/io/ebean/dbmigration/ddlgeneration/platform/BaseTableDdl.java

```diff
@@ -970,17 +993,20 @@ protected void addUniqueConstraint(DdlWrite writer, AlterColumn alter, String uq
 
 
   protected void alterTableDropColumn(DdlBuffer buffer, String tableName, String columnName) throws IOException {
-
-    buffer.append("alter table ").append(tableName).append(" drop column ").append(columnName)
-      .endOfStatement();
+    platformDdl.alterTableDropColumn(buffer, tableName, columnName);
```

Comment:
dropColumn is now handled by platformDdl, because for SqlServer we must delete all refering objects before we can delete the column

Reply:
Good.